### ISSUE TITLE
Riki's tricks of the trade and stop command

### ISSUE THREAD

**Issue by hondacbr600** [ValveSoftware/Dota2-Gameplay#7311]
I have enabled the option to require a stop command when channeling. I'm not sure if Tricks of the trade is considered channeled but it requires a stop command to cancel it early. With the new quick cast customization (finally, I was waiting for so long for it) I can set it to be cast on key release for better aiming, but this method is kinda slow if you want to dodge projectiles, stuns, etc. The solution is to self-cast it (alt + e), it works like a charm, except when you self-cast it you can exit out of it with a move command (right mouse click), without a stop command.

TL;DR - Riki's Tricks of the trade requires a stop command when cast "normally", but it doesn't when is self-cast

**Comment by Avarise831** [comment]
#1173
